conf/solr/8.8.1/updateSchemaMDB.sh

```diff
@@ -56,27 +56,67 @@ if [ ! -z "${UNBLOCK_KEY}" ]; then
   fi
 fi
 
+# Check if schema.xml file exists
+if [ ! -f "${TARGET}/schema.xml" ]; then
+  echo "'schema.xml' file not found in target dir '${TARGET}'"
+  usage
+  exit 0
+fi
+
 ### Retrieval
 echo "Retrieve schema data from ${DATAVERSE_URL}/api/admin/index/solr/schema"
 TMPFILE=`mktemp`
 curl -f -sS "${DATAVERSE_URL}/api/admin/index/solr/schema${UNBLOCK_KEY}" > $TMPFILE
 
 ### Fail gracefull if Dataverse is not ready yet.
-if [[ "`wc -l ${TMPFILE}`" < "3" ]]; then
+if [[ `wc -l < ${TMPFILE}` -lt 3 ]]; then
   echo "Dataverse responded with empty file. When running on K8s: did you bootstrap yet?"
   exit 123
 fi
 
-### Processing
-echo "Writing ${TARGET}/schema_dv_mdb_fields.xml"
-echo "<fields>" > ${TARGET}/schema_dv_mdb_fields.xml
-cat ${TMPFILE} | grep ".*<field" >> ${TARGET}/schema_dv_mdb_fields.xml
-echo "</fields>" >> ${TARGET}/schema_dv_mdb_fields.xml
+### Processing field entries
+echo "Replacing field lines"
+START='<!-- Dynamic Dataverse fields from http://localhost:8080/api/admin/index/solr/schema -->'
+INCL='<xi:include href="schema_dv_mdb_fields.xml" xmlns:xi="http://www.w3.org/2001/XInclude" />'
+END='<!-- End of Dynamic Dataverse fields -->'
+
+### -- create temp file with new content
+TMPF=`mktemp`
+echo "$START" > ${TMPF}
+cat ${TMPFILE} | grep ".*<field" >> ${TMPF}
```

Comment:
I don't think this is necessary anymore. Let's just save the filtered multiline string in a var to be used in the replacement text of `sed`